app/db/models/history.py

```diff
@@ -0,0 +1,15 @@
+from typing import Optional
+from sqlmodel import Field, Relationship, SQLModel
+from .common import TimestampModel
+from .user import User
+
+class History(SQLModel, table= True):
+    __tablename__ = "history"
+    
+    id: Optional[int] = Field(default=None, primary_key=True)
+    input: str
+    result: str
```

Comment:
See the definition for the [search endpoint here](https://github.com/a11rew/vip-service-backend/blob/616256c9d6934f9235960b44391441aa3729eea1/app/api/search/views.py#L9)

We accept name, gender, occupation, and age and return the same keys but with an is_vip property included. 

All of those values would not fit in a single str field no?